### PR TITLE
Link fixes4

### PR DESCRIPTION
**Pull request by TEChopra1000**
A new round of link fixes.

### CODE REVIEW

File: julia/docs/src/tutorial/char-lstm.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ networks yet, the example shown here is an implementation of LSTM by
 using the default FeedForward model via explicitly unfolding over time.
 We will be using fixed-length input sequence for training. The code is
 adapted from the [char-rnn example for MXNet's Python
-binding](https://github.com/dmlc/mxnet-notebooks/blob/master/python/tutorials/char_lstm.ipynb),
+binding](/api/r/docs/tutorials/char_rnn_model),
```
Comment (iblis17) [inline julia/docs/src/tutorial/char-lstm.md]: ah, is there a Python tutorial on the web site?

Comment (TEChopra1000) [reply julia/docs/src/tutorial/char-lstm.md]: @iblis17 there is this one: https://mxnet.apache.org/api/r/docs/tutorials/char_rnn_model.

File: docs/python_docs/python/tutorials/index.rst [hunk 2]
```diff
@@ -56,13 +56,13 @@ Packages & Modules
 
    .. card::
       :title: Symbol API
-      :link: packages/symbol/index.html
+      :link: /api/python/docs/api/symbol/index.html
 
-      How to use MXNet's Symbol API.
+      MXNet Symbol API has been depricated. API documentation is still available for reference.
```
Comment (TEChopra1000) [inline docs/python_docs/python/tutorials/index.rst]:
```suggestion
      MXNet Symbol API has been depricated. API documentation is still available for reference.
```

File: julia/docs/src/tutorial/char-lstm.md (hunk 1 above)
Comment (iblis17) [reply julia/docs/src/tutorial/char-lstm.md]: well, I think it's R's, not Python.

Comment (TEChopra1000) [reply julia/docs/src/tutorial/char-lstm.md]: Ah I missed that. What about this? https://github.com/apache/incubator-mxnet/blob/8004a027ad6a73f8f6eae102de8d249fbdfb9a2d/example/rnn/old/char-rnn.ipynb

Comment (iblis17) [reply julia/docs/src/tutorial/char-lstm.md]: Ah, nice, that's the correct one.

**Comment by ChaiBapchya** [comment]
@aaronmarkham @TEChopra1000 
The reason why you were facing MKLDNN lib error was 
Your branch was likely not updated with latest master

Your error - MKLDNN lib not found - https://github.com/apache/incubator-mxnet/issues/16629
Fixed in this PR - https://github.com/apache/incubator-mxnet/pull/16690
Needs to be included in your branch.

Steps to merge
```
git remote add upstream https://github.com/apache/incubator-mxnet

git checkout master
git fetch upstream master
git merge upstream/master
git push origin master
```

Hope this helps.

**Comment by TEChopra1000** [comment]
@aaronmarkham would you be willing to merge these link fixes?

File: docs/python_docs/python/tutorials/getting-started/gluon_from_experiment_to_deployment.md [hunk 3]
```diff
@@ -267,8 +267,7 @@ finetune_net.export("flower-recognition", epoch=epochs)
 MXNet provides various useful tools and interfaces for deploying your model for inference. For example, you can use [MXNet Model Server](https://github.com/awslabs/mxnet-model-server) to start a service and host your trained model easily.
 Besides that, you can also use MXNet's different language APIs to integrate your model with your existing service. We provide [Python](/api/python.html),    [Java](/api/java.html), [Scala](/api/scala.html), and [C++](/api/cpp) APIs.
 
-Here we will briefly introduce how to run inference using Module API in Python. There is more detailed explanation available in the [Predict Image Tutorial](https://mxnet.apache.org/tutorials/python/predict_image.html).
-In general, prediction consists of the following steps:
+Here we will briefly introduce how to run inference using Module API in Python. In general, prediction consists of the following steps:
```
Comment (ys2843) [inline docs/python_docs/python/tutorials/getting-started/gluon_from_experiment_to_deployment.md:270]: This is the broken link discussed in #16724

Comment (ChaiBapchya) [reply docs/python_docs/python/tutorials/getting-started/gluon_from_experiment_to_deployment.md:270]: Do you want to add it back? or the image-classification shouldn't exist anymore? What's the solution for that issue?

Comment (ys2843) [reply docs/python_docs/python/tutorials/getting-started/gluon_from_experiment_to_deployment.md:270]: @ChaiBapchya #16724 is still open, I wondered if something is pending. I personally don't know the expect solution, if everything is good we can close it.

But following link was mentioned in the issue, not sure if this need to be considered.
> True link should be -- https://github.com/dmlc/mxnet-notebooks/blob/master/python/tutorials/predict_imagenet.ipynb

Comment (ChaiBapchya) [reply docs/python_docs/python/tutorials/getting-started/gluon_from_experiment_to_deployment.md:270]: I see there was a link which previously existed. Now the link has been removed. Have a word with Aaron/Talia they have a better idea.

**Pull request merged**